docs/index.asciidoc

```diff
@@ -149,6 +150,15 @@ Example:
 
 NOTE: It is an error to specify both `dictionary` and `dictionary_path`.
 
+[id="plugins-{type}s-{plugin}-dictionary_file_max_bytes"]
+===== `dictionary_file_max_bytes`
```

Comment:
we should be clearer about what is the meaning of this limit. The limit applies to codepoints. Each codepoint may be 1,2 ,3 or more bytes depending on encoding and character size in the encoding. For example:

```
# creating a parser and setting code point limit to 100
> @parser = Psych::Parser.new(Psych::TreeBuilder.new)
 => #<Psych::Parser:0x407858c5 @handler=#<Psych::TreeBuilder:0x30075210 @start_line=nil, @start_column=nil, @stack=[], @end_column=nil, @root=nil, @end_line=nil, @last=nil>, @external_encoding=0> 
> @parser.code_point_limit = 100
 => 100 

# creating a 300 byte document with 100 3-byte japanese characters
> IO.write("string.yml", "車"*100); IO.read("string.yml").bytes.size
 => 300 

#  loading 300 bytes (100 code points) without errors from the parser
> @parser.parse(IO.read("string.yml")).handler.root.to_ruby.first.size
 => 100 
> @parser.parse(IO.read("string.yml")).handler.root.to_ruby.first.bytes.size
 => 300 
```

Reply:
updated as per the suggestion